### PR TITLE
B1 home page

### PR DESCRIPTION
**Pull request by sumitk-singh**
## Created the home page for video library web app.
please review it.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@500&family=Roboto:wght@500&display=swap');
+*{
+    font-family: 'Libre Franklin', sans-serif;
+    font-family: 'Roboto', sans-serif;
+}
+
+header,section{
+    margin-left: 1rem;
```
Comment (parulgupta2001) [inline src/App.css:8]: use shorthand property

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
+import './App.css';
+import { Routes, Route } from "react-router-dom";
+
+import Navbar from './components/Navbar/Navbar';
```
Comment (parulgupta2001) [inline src/App.js:5]: import components from one file like index.js

File: src/components/Card/PlaylistCard/PlaylistCard.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    --color-dark:#000000;
+}
+
+.playlist-card-main-container{
+    display: flex;
+    justify-content: space-between;
+    width: 15rem;
+    border: solid 1px;
+    padding: 10px;
+    line-height: 0px;
```
Comment (parulgupta2001) [inline src/components/Card/PlaylistCard/PlaylistCard.css:13]: no need to use px or rem after zero

File: src/components/Card/VideoCategoryCard/VideoCategoryCard.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
+:root {
+    --dark-black: #000000;
+    --light: #fcfcfc;
+    --light-gray-card: #e6e6e6dc;
+    --theam-color-primary: #F50057;
```
Comment (parulgupta2001) [inline src/components/Card/VideoCategoryCard/VideoCategoryCard.css:5]: I think you want to write "theme" so please correct the spelling

File: src/components/Footer/Footer.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
+:root {
+    --light-gray: #c6c6c6;
+}
+
+body{
+    margin-left: 0;
```
Comment (parulgupta2001) [inline src/components/Footer/Footer.css:6]: use shorthand property

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+/* body {
+  margin: 0;
```
Comment (parulgupta2001) [inline src/index.css:2]: remove commented code

File: src/pages/Home/Home.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    display: flex;
+    gap: 5rem;
+}
+
+.section-1 {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    color: var(--color-dark);
+    margin-top: 2rem;
```
Comment (parulgupta2001) [inline src/pages/Home/Home.css:18]: use shorthand property

File: src/pages/Home/Home.css [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+   
+}
+
+
+.section-2 {
+    display: flex;
+    flex: wrap;
+    justify-content: center;
+    gap: 2rem;
+    margin-left: 2rem;
```
Comment (parulgupta2001) [inline src/pages/Home/Home.css:35]: here also use shorthand property

File: src/pages/Home/Home.jsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+              <h2 className='txt-big'>Fitness start here</h2>
+             
+              <q>
+              If you want something you’ve never had, you must be willing to do something you’ve never done.
+              </q>
+              <p>Let's start today</p>
+              <Link to='/explore'> <button className="btn-section-1" >Explore</button></Link>
+            </div>
+            <div>
+              <img className="home-page-img" src="/assets/home-page-image.png" alt="" />
```
Comment (parulgupta2001) [inline src/pages/Home/Home.jsx:31]: do not left alt blankl

**Review by parulgupta2001 — APPROVED** [review]
Do some changes as suggested otherwise everything looks good.

**Review by Shubhamkr2610 — APPROVED** [review]
do correction that other people have suggested , else everything looks good to me

File: src/App.js (hunk 2 above)
Comment (sumitk-singh) [reply src/App.js:5]: ok

**Pull request merged**